eng/PatchConfig.props

```diff
@@ -100,4 +100,8 @@ Later on, this will be checked using this condition:
       @aspnet/signalr-protocol-msgpack;
     </PackagesInPatch>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.28' ">
```

Comment:
Above changes are for `2.1.26` and this is `.28` is that intentional?

Reply:
Ah so this is the `AspNetCore` package version https://github.com/dotnet/aspnetcore/pull/31547/files#diff-e4bed5b736f205989dd4fdb6d78acfe9126577983d325d378ce91794d74e63c8R5

Reply:
Yes, my changes in build/dependencies.props are just to use the packages that are already public in rolling and PR builds. We'll move to 2.1.27 when updating the baselines on Patch Tuesday.